601-spring-data-primitive-types/src/main/java/io/quarkus/qe/spring/data/model/Animal.java

```diff
@@ -0,0 +1,40 @@
+package io.quarkus.qe.spring.data.model;
+
+import java.time.LocalDateTime;
+import javax.persistence.Column;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.MappedSuperclass;
+
+// issue QUARKUS-525
+@MappedSuperclass
+public class Animal {
+
+    @Id
+    @GeneratedValue
+    public long id;
+
+    @Column(columnDefinition = "TIMESTAMP")
+    public LocalDateTime birthDay;
+
+    @Column(columnDefinition = "TIMESTAMP")
+    public LocalDateTime died;
+
+    public String deathReason;
+
+    public long getId() {
+        return id;
+    }
+
+    public LocalDateTime getBirthDay() {
+        return birthDay;
+    }
+
+    public LocalDateTime getDied() {
+        return died;
+    }
+
+    public String getDeathReason() {
```

Comment:
Do we need the getter methods? I tried with getters and without, and the test passes.
Also, take into account that the rest of entities define the fields as private fields and then the getter and the setter methods.